src/index.js

```diff
@@ -637,6 +697,21 @@ function wrapIndyCallWithErrorHandling(func) {
         throw e
       }
 
+      if (Platform.OS === 'ios') {
```

Comment:
We could maybe remove this condition and it still should work. @TimoGlastra what do you think? Then we can remove the logic from Android eventually or leave it until it really bothers us.

Reply:
If it works for both Android and iOS, I suggest to remove the condition